### PR TITLE
[TRUNK-5297] Delete redundant type cast

### PR DESCRIPTION
**Pull request by BartlomiejRasztabiga**
Issue: https://issues.openmrs.org/browse/TRUNK-5297

### CODE REVIEW

**Comment by dkayiwa** [comment]
@BartlomiejRasztabiga we have some merge conflicts here.

**Comment by BartlomiejRasztabiga** [comment]
@dkayiwa Resolved

**Comment by BartlomiejRasztabiga** [comment]
@dkayiwa Resolved

File: api/src/main/java/org/openmrs/api/PasswordException.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public PasswordException() {
 	}
 	
 	public PasswordException(String message, Throwable cause) {
-		super(message, (Object[]) null, cause);
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/api/PasswordException.java]: Did you take a look at this? https://issues.openmrs.org/browse/TRUNK-5016

**Comment by tomekdo3** [comment]
Looks goot to me :)

File: api/src/main/java/org/openmrs/api/PasswordException.java (hunk 1 above)
Comment (dkayiwa) [reply api/src/main/java/org/openmrs/api/PasswordException.java]: @BartlomiejRasztabiga did you see the comment above?

**Comment by dkayiwa** [comment]
Closing for lack of activity.

**Pull request closed without merging**